rudder-core/src/main/scala/com/normation/rudder/services/policies/RudderPromiseWriterServiceImpl.scala

```diff
@@ -313,6 +325,31 @@ class RudderCf3PromisesFileWriterServiceImpl(
 
   }
 
+  // Write a Json file containing all properties defined on the Node so they are usable in agent code
+  private[this] def writeNodePropertiesFile (node: NodeConfiguration, outPath: String) = {
+
+    def generateNodePropertiesJson(properties : Seq[NodeProperty]): JValue = {
+      import net.liftweb.json.JsonDSL._
+      import com.normation.rudder.domain.nodes.JsonSerialisation._
+      ( "node-properties" -> properties.toDataJson())
+    }
+
+    val fileName = "node-properties.json"
+    val path = outPath + "/properties.d"
```

Comment:
You should not use "/" but FileSeparator. 
Or new File(new File(outPath, propertiesDir), fileName)